postprocessing/science/get_distances_to_rupture.jl

```diff
@@ -0,0 +1,195 @@
+# Script to calculate precise distances from SeisSol's surface/volume cells to the rupturing part of the fault
+# Output will be saved as a numpy file (.npy)
```

Comment:
why not saving as a netcdf file (if to be used with asagi)

Reply:
Why would you use the output with asagi? Numpy format can easily be converted

Reply:
why would you not spare a conversion?